flowzone.yml

```diff
@@ -582,7 +587,9 @@ jobs:
             echo "enabled=false" >> $GITHUB_OUTPUT
           fi
 
-      - name: Check for custom jobs
+      - name: Check for custom actions
+        # skip custom actions for external contributions by default
+        if: github.event.pull_request.head.repo.full_name == github.repository || inputs.restrict_custom_actions == false
```

Comment:
Maybe we should print a notice here instead of just quietly skipping the custom actions?